tox.ini

```diff
@@ -33,6 +33,16 @@ commands =
   python -m sphinx -v -b doctest -d doctrees docs html
   find html -type f -name "*.ipynb" -not -path "html/_sources/*" -delete
 
+[testenv:docs-clean]
+description = Clean all the existing built objects and build them from scratch
+allowlist_externals =
+  find
+  tox
+commands =
+  find {toxinidir}/doctrees ! -name doctrees -delete
+  find {toxinidir}/html ! -name html -delete
```

Comment:
What about the `generated` folder, may that need cleaning as well?

What about the `-E` option? Should that be used? See https://www.sphinx-doc.org/en/master/man/sphinx-build.html#cmdoption-sphinx-build-e.

Reply:
> What about the generated folder, may that need cleaning as well?

Yes, you're right...! I will add it.

Reply:
Actually, I think `-E` option will be enough.
I just tested updating minor things in css and then run the current version and it looks working as expected.
Or, should we keep the deleting folders part just in case...?